src/app/core/sidenav/sidenav.component.ts

```diff
@@ -56,7 +56,7 @@ export class SidenavComponent implements OnInit, OnDestroy {
 
     this.appStateService.onRecordSelected().subscribe(resp => {
       this.rowId = resp.cinchyId;
-      this.filteredTableUrl = this.tableUrl ? `${this.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${resp}` : this.filteredTableUrl;
+      this.filteredTableUrl = !this.rowId.toString().includes('null') ? `${this.tableUrl}?viewId=0&fil[Cinchy%20Id].Op=Equals&fil[Cinchy%20Id].Val=${this.rowId}` : this.tableUrl;
```

Comment:
shouldn't we check if it's null and not convert it to a string?

Reply:
@cinchy-yogeshjagtap ☝️

Reply:
@alvin-h FYI, this now works as expected. Whether or not it needs the change you mention here I'll leave to you two.